packages/code-du-travail-frontend/pages/dossiers/ministere-du-travail-notre-dossier-sur-le-coronavirus/ce-que-changent-les-ordonnances-en-droit-du-travail.js

```diff
@@ -20,11 +20,11 @@ class Fiche extends React.Component {
   static async getInitialProps({ asPath }) {
     // beware, this one is undefined when rendered server-side
     const anchor = asPath.split("#")[1];
-    return { anchor };
+    return { anchor, asPath };
```

Comment:
what do you think about translating the component into a function and use the useRouter Hook to retrieve the `asPath` value

Reply:
100% ok with that